### PR TITLE
Add image format disclaimer to PWA icon labels

### PR DESCRIPTION
**Pull request by victorhmp**
#### What problem is this solving?

JPEG images being uploaded as icons to be used by our PWA solution would result in unwanted behavior. This PR simply adds a disclaimer to let users know that they should upload their icons as png files.

#### How should this be manually tested?

[Workspace](https://victorvtex--corona.myvtex.com/admin/cms/store)

#### Screenshots or example usage

Screen Shot 2021-03-30 at 09 11 11

#### Type of changes

| ✔️  | Type of Change                                                                            |
| --- | ----------------------------------------------------------------------------------------- |
| \_  | Bug fix                                |
| ✔️  | New feature                        |
| \_  | Breaking change  |
| \_  | Technical improvements  |

### CODE REVIEW

File: messages/pt.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "admin/store-name.description": "This is your store's name, in Portuguese",
```
Comment (klzns) [inline messages/pt.json:2]: Is that expected? 🤔 

```suggestion
  "admin/store-name.description": "This is your store's name, in Portuguese",
```

File: messages/es.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  "admin/store-name.description": "This is your store's name, in Spanish",
```
Comment (klzns) [inline messages/es.json:2]: same

**Review by klzns — CHANGES_REQUESTED** [review]

File: messages/pt.json (hunk 1 above)
Comment (victorhmp) [reply messages/pt.json:2]: Whaaaaat?? I don't know where this message came from 🤔

Comment (victorhmp) [reply messages/pt.json]: Removed it everywhere

**Review by klzns — APPROVED** [review]

**Pull request merged**